util/rlp/fuzz/gen_untrusted_corpus.rs

```diff
@@ -0,0 +1,113 @@
+extern crate rlp;
+extern crate ethcore_bigint;
+
+use std::env;
+use std::error::Error;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::fs::File;
+use rlp::RlpStream;
+use ethcore_bigint::prelude::{U128, U256, H64, H128, H160, H256, H512, H520, H2048};
+
+fn create_file(path: &PathBuf) -> File {
+    match File::create(path.as_path()) {
+        Err(why) => panic!("couldn't create {}: {}",
+                           path.display(),
+                           why.description()),
+        Ok(file) => file
+    } 
+}
+
+fn write_to_file(f: &File, rlp: &RlpStream) {
+    let mut g = f.clone();
+    match g.write_all(rlp.as_raw()) {
+        Err(why) => {
+            panic!("couldn't write to file: {}", why.description())
+        },
+        Ok(_) => println!("successfully wrote to file")
+    }
+
+}
+
+fn write_corpus(corpdir: &str, rlp: &RlpStream) {
+    // Read in base path to fuzzing corpus directory from `RLPCORPUS` environment var
+    let corp = env::var("RLPCORPUS").unwrap();
```

Comment:
what is fuzzing corpus? does it need to be persistent? also:

1. It will crash on everyone's machine, cause `RLPCORPUS` is no defined
2. I see no reason for tests to spoil my filesystem.